Temperature_API/schemas.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class TemperatureBase(BaseModel):
+    date_time: datetime
+    temperature: float
```

Comment:
The `temperature` field is defined as a float, which is appropriate for temperature values. Ensure that any input data is validated to prevent errors during data processing.